### PR TITLE
Dropping dependency on logan square.

### PR DESCRIPTION
**Pull request by lassem**
As discussed, here is the suggestion for dropping logan square.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,9 +2,7 @@
 
 FINNFlow is FINN's additions to square's flow. It's built on top of flow, and adds whatever we've needed.
 
-**Warning 1** : This is currently fairly hackish, and will remain so until square releases flow 0.9. There are some API breaking changes in there, and we're not going to clean this up until then. At that point we'll also consider PR's to get any features we need back into flow.
```
Comment (andaag) [inline README.md:5]: Remove the "our flow migration" info as well, we've torn out that code long ago and it never was terribly good.

Comment (andaag) [inline README.md:5]: aaand, you already did that.. :p

File: finnflow/src/main/java/no/finn/android/finnflow/FinnFlow.java [hunk 2]
```diff
@@ -21,30 +21,28 @@
     Screen destinationScreen = null;
     boolean isResumed = false;
 
-    public static Intent getIntent(Context context, Screen screen, Class<?> cls) {
-        return getIntent(context, History.single(screen), cls);
+    public FinnFlow(Activity activity, boolean debugBuild) {
```
Comment (andaag) [inline finnflow/src/main/java/no/finn/android/finnflow/FinnFlow.java:25]: when initializing this in our code make sure you use FinnEnvironment.isDebug(). If we initialize this in a library we get the library's build config, not the "consumer" of the library.

File: finnflow/src/main/java/no/finn/android/finnflow/ParcelableParcer.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+public class ParcelableParcer implements flow.StateParceler {
+    @Override
+    public Parcelable wrap(Object instance) {
+        return (Parcelable) instance;
+    }
+
+    @Override
+    public Object unwrap(Parcelable parcelable) {
+        return parcelable;
```
Comment (andaag) [inline finnflow/src/main/java/no/finn/android/finnflow/ParcelableParcer.java:13]: Should atleast leave a comment here about how to use a wrapper to support for example jackson/gson etc? Can contain just example code for serialization, but it should be mentioned how this is easy to do.

File: sample/src/main/java/no/finn/android/finnflow/sample/BaseActivity.java [hunk 4]
```diff
@@ -10,4 +12,9 @@
     protected Screen createDefaultScreen() {
         return new MasterScreen();
     }
+
+    @Override
+    protected StateParceler getParcer() {
```
Comment (andaag) [inline sample/src/main/java/no/finn/android/finnflow/sample/BaseActivity.java]: You create a new parcer here every time you trigger it?

Maybe instead have a getParcer in the main code, and rename this to createParcer. Then the "base" code can contain logic for keeping the reference and whoever implements this doesn't need to worry about it.

**Comment by andaag** [comment]
Minor changes, other than that :+1:

File: sample/src/main/java/no/finn/android/finnflow/sample/BaseActivity.java (hunk 4 above)
Comment (lassem) [inline sample/src/main/java/no/finn/android/finnflow/sample/BaseActivity.java]: Fixed.

File: finnflow/src/main/java/no/finn/android/finnflow/FinnFlow.java (hunk 2 above)
Comment (lassem) [inline finnflow/src/main/java/no/finn/android/finnflow/FinnFlow.java:25]: Done.

File: finnflow/src/main/java/no/finn/android/finnflow/ParcelableParcer.java (hunk 3 above)
Comment (lassem) [inline finnflow/src/main/java/no/finn/android/finnflow/ParcelableParcer.java:13]: Added jackson to the sample app.

**Pull request merged**